index.js

```diff
@@ -552,9 +553,9 @@ class XiaomiRoborockVacuum {
     try {
       const map = await this.device.call('get_room_mapping');
       this.log.info(`INF getRoomMap | ${this.model} | Map is ${map}`);
```

Comment:
Actually, when are we calling the `getRoomMap()` method?
It's never called. The new options defined are never used xD

Reply:
sure, it is cvalled if you add the config : "autoroom".

it is not in the readme yes ... it was a first try and many people don't know how to get the IDs.
so i think will be good to let them take it out of the log.

i see a good point here https://github.com/nicoh88/homebridge-xiaomi-roborock-vacuum/issues/114

i changed the code so its called when configured AND device is discovered

Reply:
> sure, it is cvalled if you add the config : "autoroom".

Can you point me at the line of code in index.js where you check this config and call the method? I can't seem to find it... 🤔

Reply:
Oh! just saw your latest commit 👍